Mulligan.1.html

```diff
@@ -0,0 +1,174 @@
+<html>
+    <head>
+            <style>
+                 div.padded {  
+      padding-top: 50px;  
+      padding-right: 400px;  
+      padding-bottom: 0.25in;  
+      padding-left: 400px;  
+    }
+            #rcorners2 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: DodgerBlue;
+            float: left;
+            margin-right: 20px
+            
+                    }
+            #rcorners3 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: plum;
+            float: right;
+            margin-left: 20px
+                    }   
+            #rcorners4 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: orange;
+            float: left;
+            margin-right: 20px
+                    }        
+            #rcorners1 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color:red;
+            float: right;
+            margin-left: 20px
+                    }          
+
+                    .main {
+             color: black;
+             font-size: 16px;
+             font-family: Helvetica;
+             
+             }
+
+            .name {
+             color: dodgerblue;
+             font-size: 30px;
+             font-family: Helvetica;
+             
+             }
+            .name2 {
+             color: plum;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name3 {
+             color:orange;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name4 {
+             color: red;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+
+             img.rounded-corners {
+border-radius: 15px;
+}
+
+             .image-container {
+  display: flex;
+  justify-content: center;
+}
+             
+        
+            
+            </style>
+    </head>
+        <body>
+                
+                <!-- get the header image to center on the page -->
+                <div class="image-container"><img src="TMNTHead.png" alt="TMNTHead" style="width:1400px;height:832px;"></div>
+            
+            <div class="padded">
+            <!-- see if you can get this a little bit narrower so that the text wraps to more lines to match the design a little closer -->
+            
+            <p style="font-family:Helvetica;">
```

Comment:
Refrain from using inline styles. If you're going to style a paragraph, give it a class or ID and put it in your stylesheet with a declaration. Alternatively, if Helvetica is the global font, define it somewhere like `body {}`

```
.title {
 font-family: helvetica;
}
```

```
body {
     font-family: helvetica !default;
     font-size: 12px !default;
}
```

Read: https://css-tricks.com/using-default-parameters-es6/